example/android/app/src/main/AndroidManifest.xml

```diff
@@ -40,5 +39,10 @@
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+        <service android:name="ly.count.dart.countly_flutter.CountlyMessagingService">
```

Comment:
if we already have a service on SDK side then we don't need to add it to the example app.